shared/domains/chelonia/chelonia.js

```diff
@@ -567,6 +565,26 @@ export default (sbp('sbp/selectors/register', {
     // calling this will make pubsub unsubscribe for events on `contractID`
     sbp('okTurtles.events/emit', CONTRACTS_MODIFIED, state.contracts)
   },
+  'chelonia/contract/disconnect': async function (contractID, contractIDToDisconnect) {
+    const state = sbp(this.config.stateSelector)
+    const contractState = state[contractID]
+
+    const keyIds = Object.values(contractState._vm.authorizedKeys).filter((k) => {
+      // $FlowFixMe
+      return k._notAfterHeight == null && k.meta?.keyRequest?.contractID === contractIDToDisconnect
+    }).map(k => (k: any).id)
+
+    console.log('chelonia/contract/disconnect:  ', { keyIds })
+
+    if (!keyIds.length) return
```

Comment:
Is this a silent error? (All errors should be logged, exceptions thrown, etc.)

Reply:
This isn't an error per se. If you call disconnect but there is nothing to disconnect, it returns (this ensures idempotency).